slurm/memory-hog.py

```diff
@@ -1,42 +1,57 @@
 #!/usr/bin/env python3
-from __future__ import print_function, division
```

Comment:
Cause python2 is EOL? :)

Reply:
It's python2 compatible though. I was all the while using both Python2 and Python3 to ensure.
For some reason this happens:

[narimah1@login2]~% python
Python 2.7.5 (default, Aug  7 2019, 00:51:29) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-39)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> 
>>> print("hello")
hello
>>> print "hello"  
hello
>>>

Reply:
heh @ python2 is EOL... five years ago, I wouldn't have thought that we would almost be rid of python2, but we actually are... anyway, glad you tested with both!  (I still try to have my most important things compatible with both).  Anyway, good that you tested!

But, printing working in the second case (and first) is sort of an accident, since it interprets it as `print` and a tuple (or in the first case, parentheses are grouping symbols).  So I'd recommend putting it back.

(`division` wasn't needed here, I tend to put in division and print_statement at the top of all my code, just to ensure I don't accidentally kill myself later...)